src/verify-and-receive.ts

```diff
@@ -15,7 +15,9 @@ export async function verifyAndReceive(
   // verify will validate that the secret is not undefined
   const matchesSignature = await verify(
     state.secret,
-    event.payload,
+    typeof event.payload === "string"
+      ? event.payload
+      : JSON.stringify(event.payload),
```

Comment:
This could cause problems if GitHub ever changes payload serialization, so the signature would not match...

We should check if it's possible to disable this parsing in Vercel.

Reply:
It is possible https://github.com/probot/probot/issues/1955#issuecomment-1919925057

Reply:
Seems like you don't even need to do that: https://vercel.com/guides/how-do-i-get-the-raw-body-of-a-serverless-function

Reply:
https://github.com/octokit/webhooks.js/pull/963 should be a better implementation based on that guide

Reply:
I have no preference. But you should add a test to your PR.